tasklogger/logger.py

```diff
@@ -75,7 +75,7 @@ def __init__(
         self.tasks = {}
         self.name = name
         self.min_runtime = min_runtime
-        self.stream = stream
+        self.stream = self.__parse_stream__(stream)
```

Comment:
Curious why a) a static method and not just a function, and b) why the dunder and not just a regular private method?